### DISCUSSION TITLE
Org agenda tags go outside the buffer

### DISCUSSION THREAD

**Discussion by connormclaud** [minad/org-modern#207] [Ideas]
I have the following issue when org agenda tags are shown with `auto` setting (aligned on the right)
![image](https://github.com/minad/org-modern/assets/849162/b033c1b2-ffd1-4805-ac84-cab356acbadd)
It goes beyond the buffer boundaries. 

I believe it is related to tag inheritance when tags are separated not with one colon symbol but several.
![image](https://github.com/minad/org-modern/assets/849162/de95f011-f641-4bd2-9871-7ec36bb3d177)

**Comment by minad** [comment]
org-modern is not compatible with tag auto alignment. If you like, you can provide a patch which adds this functionality.

**Comment by connormclaud** [comment]
I've managed to right justify tags with some quick code. PR will follow shortly

![image](https://github.com/minad/org-modern/assets/849162/55b156bb-52f2-43c3-a3fe-79450919d71d)

**Reply by connormclaud** [reply]
PR https://github.com/minad/org-modern/pull/209 is ready
